README.md

```diff
@@ -1,12 +1,44 @@
-# How to write exercises for the learn-ocaml platform? (Resources)
+# Step 3: Grading with generators for Ocaml built-in types
 
 This repository contains the source files for the tutorial
 
 [How to write exercises for the learn-ocaml platform?](https://github.com/ocaml-sf/learn-ocaml/blob/master/docs/howto-write-exercises.md)
 
-To get the source files for step X do:
+To go to another step and get the corresponding source files do (for step X):
 ```
 git checkout step-X
 ```
 
-For instance, for step 1, do `git checkout step-1`.
\ No newline at end of file
+## What is to learn in this step ?
+
+
+* how to use a predefined sampler
```

Comment:
In enumeration, when the item is a sentence, please use an uppercase and a final dot.